scipy/optimize/meson.build

```diff
@@ -106,7 +106,7 @@ py3.extension_module('_slsqp',
   [f2py_gen.process('slsqp/slsqp.pyf'), 'slsqp/slsqp_optmz.f'],
   fortran_args: fortran_ignore_warnings,
   link_args: version_link_args,
-  dependencies: [fortranobject_dep],
+  dependencies: [lapack_dep, blas_dep, fortranobject_dep],
```

Comment:
Is this actually needed, or is that a left-over from debugging?

Reply:
Ah! yes. Debugging left over.

Reply:
```suggestion
  dependencies: [fortranobject_dep],
```